### PR TITLE
Adds SurBTC Market to the application

### PR DESCRIPTION
**Pull request by theWando**
Implemented the necessary API call to the SurBTC provider

### CODE REVIEW

File: dataModule/src/main/java/com/mobnetic/coinguardian/model/market/SurBTC.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        });
+    }
+
+    public SurBTC() {
+        super(NAME, TTS_NAME, CURRENCY_PAIRS);
+    }
+
+    @Override
+    public String getUrl(int requestId, CheckerInfo checkerInfo) {
+        return String.format(URL, checkerInfo.getCurrencyBase().toLowerCase(), checkerInfo.getCurrencyCounter().toLowerCase());
```
Comment (MaciejCiemiega) [inline dataModule/src/main/java/com/mobnetic/coinguardian/model/market/SurBTC.java]: Please use `getCurrencyBaseLowerCase()` and `getCurrencyCounterLowerCase()` on `checkerInfo`

**Comment by theWando** [comment]
done making the changes requested

**Comment by theWando** [comment]
I'll contact the surbtc team to see what is up witth the /markets api

**Comment by theWando** [comment]
Haven't had any changes in their api

**Comment by theWando** [comment]
It does now implement dynamic currency pairs fetching, so that must be all.

**Comment by theWando** [comment]
So... ¿what about the merge?

**Comment by theWando** [comment]
@MaciejCiemiega ¿does it needs any more changes?

**Comment by MaciejCiemiega** [comment]
Thanks, I had to do some restoration and pushed it in https://github.com/mobnetic/BitcoinChecker/commit/58816f1461ccd263a75cd15caa6f1332366b4a61

**Pull request closed without merging**